Compilation Error: "Multiple definitions of function ... first defined here"

Fixed by changing extern to static following the advice here: 
https://elinux.org/Extern_Vs_Static_Inline
---
diff --git a/arch/arm/include/asm/ftrace.h b/arch/arm/include/asm/ftrace.h
index 103f7ee..90501ce 100755
--- a/arch/arm/include/asm/ftrace.h
+++ b/arch/arm/include/asm/ftrace.h
@@ -27,7 +27,7 @@ void *return_address(unsigned int);
 
 #else
 
-extern inline void *return_address(unsigned int level)
+static inline void *return_address(unsigned int level)
 {
        return NULL;
 }
